ext/ReactantNNlibExt.jl

```diff
@@ -109,7 +109,7 @@ function NNlib.conv!(
     #! format: on
 
     padding = Reactant.MLIR.IR.DenseElementsAttribute(
-        reshape(collect(padding), (num_spatial_dims, 2))
+        reshape(collect(padding), (2, num_spatial_dims))'
```

Comment:
don't call `'`/`adjoint` because it will conjugate complex matrices
```suggestion
        transpose(reshape(collect(padding), (2, num_spatial_dims)))
```